sdk/metrics/src/test/java/io/opentelemetry/sdk/metrics/internal/descriptor/MetricDescriptorTest.java

```diff
@@ -83,7 +83,7 @@ void metricDescriptor_equals() {
                 "unit",
                 InstrumentType.COUNTER,
                 InstrumentValueType.DOUBLE,
-                Advice.create(Arrays.asList(1.0, 2.0))));
+                Advice.builder().explicitBucketBoundaries(Arrays.asList(1.0, 2.0)).build()));
```

Comment:
is this intentional change in this PR?

Reply:
Oops the build was broken on main after a bad merge and I meant to fix that in a separate PR. See #5721.